src/main/java/edu/harvard/iq/dataverse/harvest/server/OAIRecordServiceBean.java

```diff
@@ -315,16 +296,17 @@ public OAIRecord findOAIRecordBySetNameandGlobalId(String setName, String global
     
     public List<OAIRecord> findOaiRecordsBySetName(String setName, Date from, Date until, boolean excludeSet) {
                 
+        if (setName == null) {
+            setName = "";
+        }
+        
         String queryString ="SELECT object(h) from OAIRecord h where h.id is not null";
-        if (setName != null) {
             if (excludeSet) {
                 queryString += " and h.setName is not null and h.setName != '' and h.setName != :setName";
             } else {
                 queryString += " and h.setName = :setName";
             } 
```

Comment:
Should the indenting above be fixed? It now seems to be indented more than it needs to be.